src/main/java/org/broadinstitute/hellbender/utils/reference/ReferenceBases.java

```diff
@@ -0,0 +1,78 @@
+package org.broadinstitute.hellbender.utils.reference;
+
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.io.Serializable;
+import java.util.Arrays;
+
+/**
+ * ReferenceBases stores the bases of the reference genome for a particular interval.
+ */
+public class ReferenceBases implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    private final byte[] bases;
+    private final SimpleInterval interval;
+
+    public ReferenceBases( final byte[] bases, final SimpleInterval interval ) {
```

Comment:
It'd be useful to have a comment indicating that this class requires the bases to be encoded at 8 bits per base (this assumption shows up in getSubset).